src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/13.2-1.sql

```diff
@@ -0,0 +1,260 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_finish_pg_upgrade()
+    RETURNS void
+    LANGUAGE plpgsql
+    SET search_path = pg_catalog
+    AS $cppu$
+DECLARE
+    table_name regclass;
+    command text;
+    trigger_name text;
+BEGIN
+
+
```

Comment:
I believe the "14" is Postgres version 14, please add a comment on why (and what) are doing this? It's not clear, how this aggregate is related to columnar extensions?